lib/utils/razorpay-utils.d.ts

```diff
@@ -0,0 +1,56 @@
+export interface IRazorpayWebhook {
```

Comment:
In the utils file, it will be a good thing if we also define the smaller util typedefs. 

eg: 

```ts
function getDateInSecs(date: Date): number {
  return (+new Date(date))/1000
}

function normalizeDate(date: Date): number {
  return isNumber(date)? date : getDateInSecs(date)
}
```

Typing smaller utils are good way to increase overall TS coverage since those utils might be used in lot of places, and TS will be able to infer more stricter types.